.github/workflows/developers_onboarding_notification.yml

```diff
@@ -0,0 +1,116 @@
+
+              function getRandomEmoji() {
+                const emojis = ['🎉', '🚀', '💪', '🌟', '🏆', '🎊', '🔥', '👏', '🌈', '🚂'];
+                return emojis[Math.floor(Math.random() * emojis.length)];
+              }
+
+              function getMessage(count) {
+                const emoji = getRandomEmoji();
+                switch(count) {
+                  case 1:
+                    return `${emoji} **Fantastic work @${author}!** Your very first PR to ${repo} has been merged! 🎉🥳\n\n` +
+                           `You've just taken your first step into open-source, and we couldn't be happier to have you onboard. 🙌\n` +
+                           `If you're feeling adventurous, why not dive into another issue and keep contributing? The community would love to see more from you! 🚀\n\n` +
+                           `For any support, feel free to reach out to the developer onboarding lead: @${mention}. Happy coding! 👩‍💻👨‍💻`;
+                  case 2:
+                    return `${emoji} **Well done @${author}!** Two PRs merged already! 🎉🥳\n\n` +
+                           `With your second PR, you're on a roll, and your contributions are already making a difference. 🌟\n` +
+                           `Looking forward to seeing even more contributions from you. The developer onboarding lead: @${mention} is here if you need any help! Keep up the great work! 🚀`;
+                  case 3:
+                    return `${emoji} **You're on fire, @${author}!** Three PRs merged and counting! 🔥🎉\n\n` +
+                           `Your consistent contributions are truly impressive. You're becoming a valued member of our community! 💖\n` +
+                           `Have you considered taking on some more challenging issues? We'd love to see what you can do! 💪\n\n` +
+                           `Remember, @${mention} is always here to support you. Keep blazing that trail! 🚀`;
+                  case 5:
+                    return `${emoji} **High five, @${author}!** You've hit the incredible milestone of 5 merged PRs! 🖐️✨\n\n` +
+                           `Your dedication to ${repo} is outstanding. You're not just contributing code; you're shaping the future of this project! 🌠\n` +
+                           `We'd love to hear your thoughts on the project. Any ideas for new features or improvements? 🤔\n\n` +
+                           `@${mention} and the whole team applaud your efforts. You're a superstar! 🌟`;
+                  case 10:
+                    return `${emoji} **Double digits, @${author}!** 10 merged PRs is a massive achievement! 🏆🎊\n\n` +
+                           `Your impact on ${repo} is undeniable. You've become a pillar of our community! 🏛️\n` +
+                           `We'd be thrilled to have you take on a mentorship role for newer contributors. Interested? 🧑‍🏫\n\n` +
+                           `@${mention} and everyone here are in awe of your contributions. You're an open source hero! 🦸‍♀️🦸‍♂️`;
```

Comment:
`${emoji} **Double digits, @${author}!** 10 merged PRs is a massive achievement! 🏆🎊\n\n` +
                           `Your impact on ${repo} is undeniable. You've become a pillar of our community! 🏛️\n` +
                   You're a quantum hero! 🦸‍♀️🦸‍♂️`;